dashboard/src/utils/pluralize.js

```diff
@@ -0,0 +1,16 @@
+import { plural } from "pluralize";
+
+function shouldBePlural(n, { locale = "en-us" }) {
+  const rules = new Intl.PluralRules(locale);
+  const rule = rules.select(n);
+  return rule !== "one";
+}
+
+function toPlural(word, n = 2, opts = {}) {
+  if (shouldBePlural(n, opts)) {
+    return plural(word);
```

Comment:
I don't think we should depend on a dictionary based pluralization solution. It doesn't save us much hassle over manually declaring the pluralization cases where we need